portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -193,13 +193,63 @@ definition {
 	}
 
 	@description = "Verify that an option can be selected as a Predefined Value"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test PredefinedValueCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132202 PredefinedValueCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Single Selection",
+			formPageNumber = "0");
+
+		Type(
+			key_rowNumber = "1",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 1");
+
+		Type(
+			key_rowNumber = "2",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 2");
```

Comment:
Hi @leticiachiu ,
I'd like to ask you to use FormFields.editOptionsRow to edit a Single Selection option.

In this scenario, you would use something like this to edit the first option:

FormFields.editOptionsRow(			
			optionName = "Option1",			
			optionValue = "Option 1");
You can find an example here.

Reply:
Hello @brunofariasdeo, I tried to use this macro but when I use:
`FormFields.editOptionsRow(			
			optionName = "Option 1",			
			optionValue = "Option 1");

FormFields.editOptionsRow(			
			optionName = "Option 2",			
			optionValue = "Option 2");`

The first value is edited twice, but it doesn't add another field.. Do you want me to use your suggestion for Option 1 and that Type in Option 2?